portfolio/src/test/java/com/google/sps/servlets/MessageTest.java

```diff
@@ -0,0 +1,51 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public final class MessageTest {
+
+  @Test
+  public void testMessage() {
+    Message message = new Message("1", "Hi", 1);
+
+    Assert.assertEquals("1", message.userId);
+    Assert.assertEquals("Hi", message.userText);
+    Assert.assertEquals(1, message.timeCreated);
+  }
+
+  @Test
+  public void testLessThanAndGreaterThan() {
+    Message message1 = new Message("0", "a", 1);
+    Message message2 = new Message("1", "b", 2);
+
+    Assert.assertTrue(message1.compareTo(message2) > 0);
+    Assert.assertTrue(message2.compareTo(message1) < 0);
+  }
+
+@Test
+  public void testEqual() {
```

Comment:
Might be worth adding an actual 

boolean equals(message)

function rather than using compareTo as an equivalent class. Is there a natural ordering to messages? You already test compareTo above with a natural ordering. So define an equals() function and move this == 0 case to the test above.

Reply:
I just wanted to test compareTo, maybe the name of the test was confusing.

Reply:
A relatively common thing is to explicitly name the method or endpoint being tested in the name of the test, along with a brief description of what exactly is being tested.

Reply:
Also, why don't you go ahead and implement equals. Should be simple.